Makefile

```diff
@@ -75,6 +75,9 @@ test-all: quality ## run tests on every supported Python/Django combination
 
 validate: quality test ## run tests and quality checks
 
+isort:  ## fix improperly sorted imports
+	isort test_utils openedx_events manage.py setup.py
+
```

Comment:
Adding this command comes pretty handy, but these other changes make the PR difficult to review:
https://github.com/openedx/openedx-events/pull/347/files#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52

https://github.com/openedx/openedx-events/pull/347/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947

Since we have to discuss whether or not to include them, which might delay the review. Can we remove them? Thanks!

Reply:
Those both seem pretty innocuous - 120 chars is the max length proposed by the edX Python style guide: https://docs.openedx.org/en/latest/developers/references/developer_guide/style_guides/python-guidelines.html#syntax-and-organization

and we have plenty of other repos that ignore emacs backup files: https://github.com/openedx/edx-platform/blob/83af6537dd2b489f4c6e626472cc768bb2331320/.gitignore#L20

Reply:
Yes, the max length is already configured here for the quality tool we use: https://github.com/openedx/openedx-events/blob/main/pylintrc#L312; we don't use flake8. I don't have a preference about the emacs ignored files.